.github/actions/trun/main.ts

```diff
@@ -0,0 +1,90 @@
+import * as core from "./deps/actions/core.ts"
+import { PQueue } from "./deps/pqueue.ts"
+import puppeteer from "./deps/puppeteer.ts"
+import { deferred } from "./deps/std/async.ts"
+import { parse } from "./deps/std/flags.ts"
+import * as path from "./deps/std/path.ts"
+import { run, runWithBrowser, runWithDeno } from "./run.ts"
+
+const flags = parse(Deno.args, {
+  string: ["dir", "concurrency", "ignore", "importMap", "filter"],
```

Comment:
Is `dir` necessary? Why not assume cwd?

Reply:
The tool runs from the root of the capi repo unless the expectation is for it to `cd` into the examples directory

Reply:
And ideally we shouldn't need to go into the examples directory since we also specific the `import_map.json` which is in the root. hence a dir flag is necessary here.